LICENSE.md

```diff
@@ -0,0 +1,181 @@
+# License
```

Comment:
I think this mixed license is confusing, is the specification MIT or CC? Can we pick just one?

Reply:
let's go with CC - I'll amend